src/components/ContentListTable/components/StatusIcon.tsx

```diff
@@ -0,0 +1,68 @@
+import { CheckCircleIcon, ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import { red, green, yellow } from '../ContentListTable'
+
+interface Props {
+  status?: string,
+  error?: string,
+  classes?: any, // TODO: type this
+}
+
+const StatusIcon = ({ status, error, classes }: Props) => {
+  switch(status) {
+    case 'Valid':
+      return (
+        <Flex alignContent={{ default: 'alignContentCenter' }} direction={{ default: 'row' }}>
+          <FlexItem spacer={{ default: 'spacerSm' }}>
+            <CheckCircleIcon color={green} />
+          </FlexItem>
+          <FlexItem>
+            <span className={classes.green}>Valid</span>
```

Comment:
I think a better way to do this would be to first create a "green" color variable in a helper.ts, and then a component named StatusText with the span and passed in color variable.

Reply:
Make your classes within that component.